site/content/events/2016-london/_sponsors.txt

```diff
@@ -88,5 +88,5 @@ filter:    erb
 ]%>
 
 <% @ersponsors.each do |sponsor| %>
-<a href='<%= sponsor[:link] %>'><img border=0 alt='<%= sponsor[:name] %>' title='<%= sponsor[:name] %>' width=120 height=48 src='<%= File.join('/',@eventhome,"logos/#{sponsor[:image]}")%>'></a>
+<a href='<%= sponsor[:link] %>'><img border=0 alt='<%= sponsor[:name] %>' title='<%= sponsor[:name] %>' width=190 src='<%= File.join('/',@eventhome,"logos/#{sponsor[:image]}")%>'></a>
```

Comment:
Just want to verify that you meant to both increase the width *and* remove the height parameter entirely.